### PR TITLE
Erisx update

### PR DESCRIPTION
**Pull request by MikeHathaway**

### CODE REVIEW

File: market_maker_keeper/erisx_market_maker_keeper.py [hunk 1]
```diff
@@ -376,5 +379,49 @@ def place_order_function(new_order_to_be_placed):
             else:
                 logging.info(f"New {side} Order below size minimum of {min_amount}. Order of amount {amount} ignored.")
 
+    def check_cancellations(self):
+        """
+        Check to see if ErisX has cancelled any of our orders.
+        If an order has been cancelled, remove it from the order book
+        """
+        with self._lock:
```
Comment (EdNoepel) [inline market_maker_keeper/erisx_market_maker_keeper.py:387]: I only see `_lock` used in one place.  Is `synchronize_orders` called from multiple threads simultaneously?

File: market_maker_keeper/erisx_market_maker_keeper.py [hunk 2]
```diff
@@ -376,5 +379,49 @@ def place_order_function(new_order_to_be_placed):
...
+            return
+
+        # Place new orders
+        self.place_orders(bands.new_orders(our_buy_orders=self.our_buy_orders(order_book.orders),
+                                            our_sell_orders=self.our_sell_orders(order_book.orders),
+                                            our_buy_balance=self.our_available_balance(order_book.balances,
+                                                                                        self.token_buy()),
+                                            our_sell_balance=self.our_available_balance(order_book.balances,
+                                                                                        self.token_sell()),
+                                            target_price=target_price)[0])
```
Comment (EdNoepel) [inline market_maker_keeper/erisx_market_maker_keeper.py:424]: What does the array returned by `new_orders` represent, and why are we only placing orders for the first element of it?

**Review by EdNoepel — COMMENTED** [review]
Some questions within.

File: market_maker_keeper/erisx_market_maker_keeper.py (hunk 1 above)
Comment (MikeHathaway) [reply market_maker_keeper/erisx_market_maker_keeper.py:387]: Nope, that was vestigial code. I will remove it.

File: market_maker_keeper/erisx_market_maker_keeper.py (hunk 2 above)
Comment (MikeHathaway) [reply market_maker_keeper/erisx_market_maker_keeper.py:424]: `new_orders` represents the set of new orders that would comply with our bands config, and that haven't been placed already. We take the `[0]` index of `bands.new_orders` as it returns a Tuple, and we are specifically interested in the new_orders list that is the first returned. https://github.com/makerdao/market-maker-keeper/blob/master/market_maker_keeper/band.py#L325 The `missing_buy_amount` and `missing_sell_amount` Tuple returns appear to actually be dead code. Also, this logic should be consistent with other keepers.

Comment (EdNoepel) [reply market_maker_keeper/erisx_market_maker_keeper.py:424]: 👍

**Review by EdNoepel — APPROVED** [review]
LGTM 👌

**Pull request merged**